var/spack/repos/builtin/packages/suntans/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Suntans(MakefilePackage):
+    """The Stanford unstructured-grid,
+    nonhydrostatic, parallel coastal
+    ocean model. """
+
+    homepage = "https://github.com/ofringer/suntans"
+    url      = "https://github.com/ofringer/suntans/archive/master.zip"
+
+    maintainers = ['ofringer', 'zyaj', 'mrayson', 'jadelson', 'lhxone']
+
+    version('3.1', '9b5d0dee5502a5650e37bf72f64b1124be86c084e7408aa517c214ddb25e36de')
+
+    depends_on('libx11@1.6.3', type = ('build', 'run'))
+    depends_on('mpi', type = ('build', 'run'))
```

Comment:
The default build type is fine. Roughly `build` is for tools run in a separate process during build, `link` is for libraries linked to any artifact in this package, 'run' is for tools run in a separate process at runtime:
```suggestion
    depends_on('libx11@1.6.3')
    depends_on('mpi')
```

Reply:
applied changes